ts/src/bitget.ts

```diff
@@ -1395,6 +1395,7 @@ export default class bitget extends Exchange {
                 'TONCOIN': 'TON',
             },
             'options': {
+                'uta': false,
```

Comment:
@carlosmiei imo, per the behavior of `handleOptionsAndParams (..., 'fetchMarkets', 'uta',...)`, the `uta` property should live under method-related dict in options (as long as that `uta` is only used in `fetchMarkets`). but as you wish

Reply:
@ttodua that does not make sense, `UTA` is a "account mode" affects everything why would you add it manually to all affected methods?

Reply:
i haven't seen any other methods using UTA in this PR, seems i was lacking context and it's used in other methods too :) sure, it's right if so

Reply:
@ttodua we're starting with `fetchMarkets` but other methods will be added next

Reply:
sure, that makes sense, i didn't know it. (btw, where is bitget's UTA api docs?)

Reply:
this is the link thi: https://www.bitget.bike/api-doc/uta/account/Get-Account-Setting